cli/flox/src/commands/build.rs

```diff
@@ -423,59 +424,6 @@ pub(crate) enum PrefetchError {
     PrefetchFailed { flakeref: Url, stderr: String },
 }
 
-/// Derive the nixpkgs url to be used for builds.
```

Comment:
Reviewer note: I moved these functions to `flox-rust-sdk` so that they could be tested using the test helpers in that crate (which can't be used from another crate).